curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-fetch-promises-and-async-await-by-building-an-fcc-forum-leaderboard/643c8ee68f280c0a58a3d439.md

```diff
@@ -0,0 +1,157 @@
+---
+id: 643c8ee68f280c0a58a3d439
+title: Step 29
+challengeType: 0
+dashedName: step-29
+---
+
+# --description--
+
+To print the result on the page, first you need to target the HTML element with the `id` of `posts-container`. like this 
+
+```js
+const postsContainer = document.getElementById('posts-container');
```

Comment:
Let's remove the example here too because campers will be comfortable with the `getElementById ` method by the time this project comes around.

Reply:
done